### PR TITLE
plasma-infra: Improve lerna bootstrap time execution [Part 3]

### PR DESCRIPTION
**Pull request by Yakutoc**
Продолжаем ускорять. 

### "PR Documentation and Storybook"

Теперь для этого workflow **высчитываем** набор scope. 

Так же пропускаем шаги с **build** сайта документации и **storybook** для пакета, если нет соответствующего scope.   

### Change detection

Так же теперь для получения списка пакетов где были произведены изменения используем `npx lerna la --json` что бы сразу работать с JSON форматом.     

:baby_chick: Download canary assets:

[color_asdk_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_ios-swift--canary.577.5452199845.swift)  
[color_asdk_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_kotlin--canary.577.5452199845.kt)  
[color_asdk_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_react-native--canary.577.5452199845.ts)  
[color_asdk_xml--canary.577.5452199845.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_asdk_xml--canary.577.5452199845.xml)  
[color_plasma_b2c_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.577.5452199845.swift)  
[color_plasma_b2c_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.577.5452199845.kt)  
[color_plasma_b2c_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.577.5452199845.ts)  
[color_plasma_b2c_xml--canary.577.5452199845.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.577.5452199845.xml)  
[color_sberHealth_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.577.5452199845.swift)  
[color_sberHealth_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.577.5452199845.kt)  
[color_sberHealth_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.577.5452199845.ts)  
[color_sberHealth_xml--canary.577.5452199845.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.577.5452199845.xml)  
[color_sbermarket_business_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.577.5452199845.swift)  
[color_sbermarket_business_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.577.5452199845.kt)  
[color_sbermarket_business_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.577.5452199845.ts)  
[color_sbermarket_business_xml--canary.577.5452199845.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.577.5452199845.xml)  
[color_sbermarket_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.577.5452199845.swift)  
[color_sbermarket_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.577.5452199845.kt)  
[color_sbermarket_metro_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.577.5452199845.swift)  
[color_sbermarket_metro_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.577.5452199845.kt)  
[color_sbermarket_metro_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.577.5452199845.ts)  
[color_sbermarket_metro_xml--canary.577.5452199845.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.577.5452199845.xml)  
[color_sbermarket_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.577.5452199845.ts)  
[color_sbermarket_selgros_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.577.5452199845.swift)  
[color_sbermarket_selgros_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.577.5452199845.kt)  
[color_sbermarket_selgros_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.577.5452199845.ts)  
[color_sbermarket_selgros_xml--canary.577.5452199845.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.577.5452199845.xml)  
[color_sbermarket_wlbusiness_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.577.5452199845.swift)  
[color_sbermarket_wlbusiness_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.577.5452199845.kt)  
[color_sbermarket_wlbusiness_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.577.5452199845.ts)  
[color_sbermarket_wlbusiness_xml--canary.577.5452199845.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.577.5452199845.xml)  
[color_sbermarket_xml--canary.577.5452199845.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.577.5452199845.xml)  
[color_sberonline_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.577.5452199845.swift)  
[color_sberonline_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.577.5452199845.kt)  
[color_sberonline_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.577.5452199845.ts)  
[color_sberonline_xml--canary.577.5452199845.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.577.5452199845.xml)  
[color_sberprime_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.577.5452199845.swift)  
[color_sberprime_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.577.5452199845.kt)  
[color_sberprime_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.577.5452199845.ts)  
[color_sberprime_xml--canary.577.5452199845.xml](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.577.5452199845.xml)  
[shadow_sbermarket_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.577.5452199845.ts)  
[typo_mage_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.577.5452199845.swift)  
[typo_mage_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.577.5452199845.kt)  
[typo_mage_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.577.5452199845.ts)  
[typo_plasma_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.577.5452199845.swift)  
[typo_plasma_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.577.5452199845.kt)  
[typo_plasma_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.577.5452199845.ts)  
[typo_ruler_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.577.5452199845.swift)  
[typo_ruler_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.577.5452199845.kt)  
[typo_ruler_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.577.5452199845.ts)  
[typo_sage_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.577.5452199845.swift)  
[typo_sage_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.577.5452199845.kt)  
[typo_sage_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.577.5452199845.ts)  
[typo_sbermarket_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.577.5452199845.swift)  
[typo_sbermarket_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.577.5452199845.kt)  
[typo_sbermarket_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.577.5452199845.ts)  
[typo_soulmate_ios-swift--canary.577.5452199845.swift](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.577.5452199845.swift)  
[typo_soulmate_kotlin--canary.577.5452199845.kt](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.577.5452199845.kt)  
[typo_soulmate_react-native--canary.577.5452199845.ts](https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.577.5452199845.ts)

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-b2c@1.218.1-canary.577.5452199845.0
  npm install @salutejs/plasma-hope@1.218.1-canary.577.5452199845.0
  npm install @salutejs/plasma-web@1.218.1-canary.577.5452199845.0
  # or 
  yarn add @salutejs/plasma-b2c@1.218.1-canary.577.5452199845.0
  yarn add @salutejs/plasma-hope@1.218.1-canary.577.5452199845.0
  yarn add @salutejs/plasma-web@1.218.1-canary.577.5452199845.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-577/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-577/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-577/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-577/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-577/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-577/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-577/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-577/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-577/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-577/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!
  
  website: http://plasma.sberdevices.ru/pr/pr-577/
  ui storybook: http://plasma.sberdevices.ru/pr/pr-577/ui-storybook/
  web storybook: http://plasma.sberdevices.ru/pr/pr-577/web-storybook/
  b2c storybook: http://plasma.sberdevices.ru/pr/pr-577/b2c-storybook/
  temple storybook: http://plasma.sberdevices.ru/pr/pr-577/temple-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-577/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-577/b2c-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-577/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-577/b2c-storybook/

File: .github/workflows/documentation-pr.yml [hunk 1]
```diff
@@ -94,20 +106,48 @@ jobs:
...
+          HAS_PLASMA_UI: ${{ fromJSON(needs.scope.outputs.HAS_PLASMA_UI) }}
+          HAS_PLASMA_TEMPLE: ${{ fromJSON(needs.scope.outputs.HAS_PLASMA_TEMPLE) }}
         with:
           github-token: ${{secrets.GH_TOKEN}}
           script: |
-            github.issues.createComment({
-              issue_number: context.issue.number,
-              owner: context.repo.owner,
-              repo: context.repo.repo,
+            const { HAS_PLASMA_B2C, HAS_PLASMA_WEB, HAS_PLASMA_UI, HAS_PLASMA_TEMPLE } = process.env;
```
Comment (Yakutoc) [inline .github/workflows/documentation-pr.yml:118]: @Yeti-or @neretin-trike @kayman233 Что думаете? 

Но у нас все равно остаются ссылки на сайте которые ведут в никуда. 

Какие минусы видите в таком подходе?

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-577/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-577/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-577/web-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-577/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-577/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-577/web-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

File: .github/workflows/documentation-pr.yml (hunk 1 above)
Comment (Yakutoc) [reply .github/workflows/documentation-pr.yml:118]: Итог: решили что такая логика нам подходит

**Review by Yeti-or — APPROVED** [review]

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-577/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-577/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-577/web-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-577/
b2c storybook: http://plasma.sberdevices.ru/pr/pr-577/b2c-storybook/
web storybook: http://plasma.sberdevices.ru/pr/pr-577/web-storybook/

File: .github/processing-scope.js [hunk 2]
```diff
@@ -8,13 +8,52 @@ const CYPRESS_SCOPE = [
...
+
+    const HAS_DOCUMENTATION_CHANGED =
+        HAS_PLASMA_UI_DOCS || HAS_PLASMA_WEB_DOCS || HAS_PLASMA_TEMPLE_DOCS || HAS_PLASMA_WEBSITE;
+
+    const scope = packagesList.filter((item) => CYPRESS_SCOPE.includes(item)).map((item) => `"${item.split('-')[1]}"`);
+
+    const scopeFullNameList = packagesList.map((item) => `${item.split('/')[1]}`);
+
+    const scopePlasmaName =
+        (HAS_PLASMA_B2C || HAS_PLASMA_WEB) && !scopeFullNameList.includes('plasma-hope')
```
[the remark this answers is not in the document]
Comment (Yakutoc) [reply .github/processing-scope.js]: @Yeti-or Сделал проще. 

Теперь в  `processing-scope.js` только подготавливаем данные 

**output**: 

`["plasma-web", "plasma-ui", "plasma-b2c"]`

И с этими данными уже работает логика в шаге **"Computed scope**":


```yml

      - name: Computed scope
        id: scope
        uses: actions/github-script@v6
        env:
          ENUMERATION: ${{ toJSON(needs.scope.outputs.PACKAGES_ENUMERATION) }}
        with:
          result-encoding: string
          script: |
            const { ENUMERATION } = process.env;
            
            const enumeration = JSON.parse(JSON.parse(ENUMERATION));
                                    
            if (!enumeration.includes('plasma-core')) {
              enumeration.push('plasma-core')
            }
            
            if (!enumeration.includes('plasma-hope')) {
              enumeration.push('plasma-hope')
            }
            
            if (!enumeration.includes('plasma-icons')) {
              enumeration.push('plasma-icons')
            }
            
            return enumeration.join(',');  

```   

ИМХо будет гибче.

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-577/

File: .github/processing-scope.js [hunk 3]
```diff
@@ -6,15 +6,74 @@ const CYPRESS_SCOPE = [
...
-    if (!CHANGED_STATE) {
+    /**
+     *
+     * @param CHANGED_STATE Information about the changed state
+     * @param CHANGED_STATE.name The name of the package.
+     * @param CHANGED_STATE.version The version of the package.
+     * @param CHANGED_STATE.private The type of the package.
+     * @param CHANGED_STATE.location The path of the package.
+     *
+     * @example
```
Comment (Yeti-or) [inline .github/processing-scope.js:19]: 👍

**Review by Yeti-or — APPROVED** [review]

**Review by kayman233 — APPROVED** [review]

**Comment by Yeti-or** [comment]
@neretin-trike  вольем?

**Review by neretin-trike — APPROVED** [review]

**Comment by neretin-trike** [comment]
> @neretin-trike вольем?

ага, :shipit:

**Pull request merged**